.eslintrc.js

```diff
@@ -299,14 +299,8 @@ module.exports = {
       rules: {
         'react/no-unused-prop-types': 'warn',
         'react/no-unused-state': 'warn',
-        'react/jsx-boolean-value': 'warn',
-        'react/jsx-curly-brace-presence': [
-          'warn',
-          {
-            props: 'never',
-            children: 'never',
-          },
-        ],
+        'react/jsx-boolean-value': 'off',
+        'react/jsx-curly-brace-presence': 'off',
```

Comment:
We can debate this, but IMO these do not provide any meaningful value into our codebase.
Also removing these will also remove all the "autofix" issues devs receive when running `yarn lint:fix`